Gopkg.toml

```diff
@@ -75,19 +75,19 @@ required = [
 
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.11.8"
+  version = "kubernetes-1.13.4"
```

Comment:
Why do we want to upgrade to this version?

Reply:
There are multiple Kubernetes Go client APIs that have been removed/changed in the new version. Examples include `WorkQueueMetrics` and `Until` that got changed in this PR. And in general, we think it's good to bring the API version used by the operator to a more recent one.

Reply:
OK, makes sense.